proto/lbm/fswap/v1/params.proto

```diff
@@ -0,0 +1,22 @@
+
+syntax = "proto3";
+package lbm.fswap.v1;
+
+option go_package = "github.com/Finschia/finschia-sdk/x/fswap/types";
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+// Params defines the parameters for the module.
+message Params {
+  option (gogoproto.goproto_stringer) = false;
+  // upper limit in new coin that this module can swap
+  cosmos.base.v1beta1.Coin new_coin_swap_total_limit = 1
+      [(gogoproto.nullable) = false, (gogoproto.castrepeated) = "github.com/Finschia/finschia-sdk/types.Coin"];
+  // constant value representing the exchange rate, without decimal (148079656)
+  uint64 swap_rate = 2;
+  // The number shows how to get the true exchange rate by dividing swap_rate by 10 to the power
+  int32 swap_rate_decimals = 3;
```

Comment:
what is the purpose of this param?

Reply:
From @jaeseung-bae , the float64 is not well in protobuf file, so I use swap_rate & swap_rate_decimals to instead.

Reply:
Oh I thought we just need to use swap_rate. what is `swap_rate_decimals` value if we use in our case?

Reply:
In this case, swap_rate will be 148079656,  and swap_rate_decimals will be 6